agriculture_shop/data/konwledge_tour.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="knowledge_tour" model="web_tour.tour">
+        <field name="name">knowledge_tour</field>
+        <field name="sequence">2</field>
+        <field name="rainbow_man_message">Congrats, enjoy your reading! </field>
```

Comment:
For each rainbow_man_message in this PR, replace by:
<field name="rainbow_man_message">Welcome ! Happy exploring. </field>

Reply:
I'm pretty sure there is no space here: "Welcome!"